<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1259</width>
    <height>767</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Graphics</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>D:/download/wallpaperflare.com_wallpaper (1).jpg</normaloff>D:/download/wallpaperflare.com_wallpaper (1).jpg</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="2" column="0">
        <widget class="QPushButton" name="btn_sort">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>sort</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="btn_search">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>search</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="btn_rect">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>rectangle</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="btn_save">
         <property name="text">
          <string>save</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="btn_circle">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>circle</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="btn_undo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>undo</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="btn_redo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>redo</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btn_line">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>line</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="sizeConstraint">
            <enum>QLayout::SetNoConstraint</enum>
           </property>
           <item>
            <widget class="QLabel" name="label_name">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>name       </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lab_x_pos">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>x_pos</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_y_pos">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>y_pos</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lab_color">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>color</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lab_width">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>width</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_legth">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>length</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_radius">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>radius</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_x_pos_final">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>x_pos final</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_y_pos_final">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>y_pos final </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lab_outline">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Outline Size</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLineEdit" name="lineEdit_name">
             <property name="frame">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_x_pos">
             <property name="frame">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_y_pos">
             <property name="frame">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox">
             <item>
              <property name="text">
               <string>red</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>blue</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>yellow</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>cyan</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>green</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>grey</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>black</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>purple</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>pink</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>orange</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>brown</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>beige</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>maroon</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_width">
             <property name="frame">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_length">
             <property name="frame">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_radius">
             <property name="frame">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_x_pos_final">
             <property name="frame">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_y_pos_final">
             <property name="frame">
              <bool>false</bool>
             </property>
             <property name="clearButtonEnabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBox">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>40</number>
             </property>
             <property name="singleStep">
              <number>2</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>draw</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_do_search">
           <property name="text">
            <string>search</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_cancel">
           <property name="text">
            <string>cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGraphicsView" name="graphicsView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
